web/src/features/fireWeather/components/tables/SortableTableByDatetime.tsx

```diff
@@ -72,68 +82,98 @@ function SortableTableByDatetime<R extends WeatherValue>(props: Props<R>) {
     setOrder(order === 'asc' ? 'desc' : 'asc')
   }
 
+  interface TableHeaderProps {
+    title: string
+    testId?: string
+  }
+
+  const TableHeader = (tableHeaderProps: TableHeaderProps) => {
+    const tableHeaderClasses = useStyles()
+    return (
+      <Toolbar data-testid={`${tableHeaderProps.testId}-header`} disableGutters={true}>
+        <Typography className={tableHeaderClasses.title}>
+          {tableHeaderProps.title}
+        </Typography>
+        <Tooltip title={open ? 'Collapse table' : 'Expand table'}>
+          <IconButton
+            aria-label={open ? 'collapse table' : 'expand table'}
+            size="small"
+            onClick={() => setOpen(!open)}
+            data-testid={`${tableHeaderProps.testId}-collapse`}
+          >
+            <KeyboardArrowUpIcon
```

Comment:
You could render different components (`<KeyboardArrowUpIcon />` or `<KeyboardArrowDownIcon />`) based on the open flag